### ISSUE TITLE
Stormcloak weird troops/ imperial giants?

### ISSUE THREAD

**Issue by LABWAZHERE** [schofida/CWORR#51]
i have noticed you added restored the more variety to troop types. 

i do have one request. 
please make it immersively diverse. 

that's the one thing i hate about the og cwo redux was finding khajiit and orcs and elves and argornians in the stormcloaks. 

it should just be human races. also the stormcloak oathbreaker using unrelenting force? (really apollo?) that makes no sense. in lore it is explained ulfric is the only one in the stormcloaks capable of doing it as he was a part of the whitebeards.

it really breaks my immersion when i find these things

![20230418130751_1](https://user-images.githubusercontent.com/128126252/232665407-f3ed0dbb-eb58-4d3d-a6c7-8e57555872b7.jpg)
this should not exist

additionally it may be worth attempting to restore the giant recruitment quest so that both sides can have giants
![20230418133602_1](https://user-images.githubusercontent.com/128126252/232675117-4986cf8b-cb69-4496-89c7-2f2f6157ece6.jpg)
This just simply isn't a fair fight (for them ;) but seriously it isn't fair) i know there is the easter egg that's fine for offence/defence missions when you are a stormcloak not when you're an imperial

TL:DR
i hate how biased the original mod is to the Stormcloaks can we make it more immersive? if not by removing beast races at least drastically lowering their spawn rates

**Comment by LABWAZHERE** [comment]
update: strangely the battle of Whiterun is the only time the giants appear which idk if that is intentional or not

**Comment by LABWAZHERE** [comment]
okay to sum it all up, argonians only seem to appear in select battles on the stormcloak side

**Comment by LABWAZHERE** [comment]
additonally
![20230428105053_1](https://user-images.githubusercontent.com/128126252/235028886-bdb5b73c-e108-473f-a2d6-7e07688f2a38.jpg)

**Comment by schofida** [comment]
I am working on some documentation regarding this. In short, no. I am not going to take troops away. This is one of my favorite parts of the mod. But I am putting documentation together to proclaim that you and everyone else had the power to change this all along ;)

**Comment by LABWAZHERE** [comment]
Fair enough, I think it is more reasonable to just level them. I am not too bothered with the stormcloak troop types (I’ll live with them it does make it more challenging which is good) it is mainly the argonians and khajiits and elves and orcs and trolls being on the Stormcloaks that bothers me. (For the troll situation I think it should be one or the other, either trolls or giants having both is just stupidly unfair)

**Comment by schofida** [comment]
There are no trolls on the Stormcloak side. Only Giants. And the imperials have mages and lots of them which I think even the odds. (I have tested this :) )

Anyway. Please see this: https://github.com/schofida/CWORR/tree/main/cwoPatcher

The patch is here.
https://github.com/schofida/CWORR/releases/tag/cwoPatcher

Let me know what you think. I want to make sure that the instructions are clear. This allows you to create your own patch with only the troops that you want.

**Comment by LABWAZHERE** [comment]
does this work with vortex? also are the vanilla errors for the rogues still going to be patched (not outright removal but adjust their skin tone).

**Comment by ds-nscorp** [comment]
Yes! I think that the only difference versus other Mod Managers is the step to add zEdit as a tool which it looks like Vortex can do. https://wiki.nexusmods.com/index.php/Tool_Setup:_xEdit
